plugins/nodes/src/js/components/LeaderGrid.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import { FormattedMessage } from "react-intl";
+import moment from "moment";
+
+import ConfigurationMap from "#SRC/js/components/ConfigurationMap";
+import ConfigurationMapHeading from "#SRC/js/components/ConfigurationMapHeading";
+import ConfigurationMapLabel from "#SRC/js/components/ConfigurationMapLabel";
+import ConfigurationMapRow from "#SRC/js/components/ConfigurationMapRow";
+import ConfigurationMapSection from "#SRC/js/components/ConfigurationMapSection";
+import ConfigurationMapValue from "#SRC/js/components/ConfigurationMapValue";
+import Loader from "#SRC/js/components/Loader";
+
+function timeFromNow(time) {
```

Comment:
is `timeFromNow` because we need to check for `null` in `time`? I wonder what others think about this one.

Reply:
I like having this function, it's more concise.

Reply:
It is like that because it is used in two places (electedTime and startTime), also because it handles the null value.